extensions/admin-pack/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Change Log
+All notable changes to the "admin-pack" extension pack will be documented in this file.
+
+Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how to structure this file.
```

Comment:
this needs to be updated

Reply:
I'll remove this file since none of our extensions has this file